gene/query.py

```diff
@@ -94,7 +94,8 @@ def add_record(self,
             containing name of the source of the match
         """
         del item['label_and_type']
-        label_types = ['aliases', 'other_identifiers', 'previous_symbols']
+        label_types = ['aliases', 'other_identifiers', 'previous_symbols',
+                       'xrefs']
         for label_type in label_types:
             if label_type not in item.keys():
                 item[label_type] = []
```

Comment:
Would you be alright with a small change in variable name here? Something like `attribute` instead of `label`?

```suggestion
        attr_types = ['aliases', 'other_identifiers', 'previous_symbols',
                       'xrefs']
        for attr_type in attr_types:
            if attr_type not in item.keys():
                item[attr_type] = []
```